src/components/Person/Person.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.css';
+
+export function Person({ name, age, partnerName, isMarried, sex }) {
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
```

Comment:
```suggestion
        {`My name is ${name}`}
```

Just so that you know it's possible this way as well